### ISSUE TITLE
Make package compatible with {dplyr} < 1.0.0

### ISSUE THREAD

**Issue by thomas-neitmann** [atorus-research/metacore#2]
I just tried to install {metacore} into our validated R environment at Roche which still uses {dplyr} 0.8.5. Unfortunately, the installation failed.

```
Error: object ‘c_across’ is not exported by 'namespace:dplyr'
Execution halted
ERROR: lazy loading failed for package ‘metacore’
* removing ‘/home/bceuser/neitmant/admiral.roche/renv/staging/1/metacore’
```

Could you adjust your code such that it works with older dplyr versions?

I'd be happy to edit the matrix config of your `R CMD check` workflow such that it runs also on earlier R versions with older packages. I did that for {admiral} and it proved to be very valuable for ensuring that the package works on our kind of old production environment.

**Comment by mstackhouse** [comment]
@thomas-neitmann I'd be happy for that contribution from you! And the backwards compatibility is surely necessary in this environment, so we can get that updated for sure.

**Comment by mstackhouse** [comment]
@thomas-neitmann I have a fix for this in #3 - you can pull and build the [issue_2](https://github.com/atorus-research/metacore/tree/issue_2) branch if you want to try it out.

**Comment by mstackhouse** [comment]
Closed with #3
